colorfield/widgets.py

```diff
@@ -27,10 +29,18 @@ def get_context(self, name, value, attrs=None):
                 "widget": self,
                 "name": name,
                 "value": value,
+                "data_coloris": {
+                    "format": context.get("format", "hex"),
+                    "required": context.get("required", False),
+                    "clearButton": not bool(context.get("required")),
+                    "alpha": bool(context.get("alpha")),
+                    "forceAlpha": bool(context.get("alpha")),
+                    "swatches": context.get("swatches", []),
+                    "swatchesOnly": bool(context.get("swatches", []))
+                    and bool(context.get("swatches_only")),
+                },
```

Comment:
Since this dict is only used by `coloris` later, it should be encoded to `json` here.